src/display/annotation_layer.js

```diff
@@ -535,6 +544,7 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
     const fontFamily = font.loadedName ? `"${font.loadedName}", ` : "";
     const fallbackName = font.fallbackName || "Helvetica, sans-serif";
     style.fontFamily = fontFamily + fallbackName;
+    // XXX revisit fontColor. Setting style.color doesn't stick.
```

Comment:
Hm, I think this worked in the original patch. Maybe the values need to be converted to RGB first?